server/bundle.go

```diff
@@ -4,15 +4,25 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"log"
 	"net/http"
 
 	"github.com/julienschmidt/httprouter"
 
+	"github.com/ndlib/bendo/items"
 	"github.com/ndlib/bendo/store"
 )
 
 // BundleListHandler handles GET requests to "/bundle/list".
 func (s *RESTServer) BundleListHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
+	if s.TapeUsed == false {
+		w.WriteHeader(503)
+		fmt.Fprintln(w, items.ErrNoStore)
```

Comment:
Don't import `items` just to use this error. Just include a literal string "Tape access is temporarily disabled".